subsys/mgmt/fmfu/Kconfig

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menu "Full Modem Firmware Update Management(FMFU)"
+
+config MGMT_FMFU
+	bool "Serial Modem Firmware Update support"
+	depends on MCUMGR_SMP_UART
+	depends on MCUMGR
+	depends on SOC_NRF9160_SICA
```

Comment:
Would it be better to state that we depend on nrf_modem_lib in some way?

Reply:
If it's possible probably yes :+1:

Reply:
```suggestion
	depends on NRF_MODEM_LIB
```

?